Change to highlight style rather than squiggly underline

ttbt28 opened the pull request:
Updated the styling of unannotated variables to use a highlighting effect instead of the previous red squiggly underline. This enhances visibility and clarity for identifying unannotated variables within the editor.

Edih112 commented on src/language-handler/model-semantic-checker/ModelSemanticChecker.ts:
```diff
@@ -6,13 +6,14 @@ import {
+import { SemanticVisitor } from ".././SemanticVisitor";
+import { ErrorUnderline, SrcPosition, SrcRange, isSubtTypeOf, varTypes } from ".././Types";
+import { Variable } from ".././Variable";
+import "./ModelSemanticChecker.css"
```
Not too important but worth noting that this css file import will cause some of the tests to fail when running `npm run test`. Removing the import for testing will fix the issue but might be worth looking into later.

Edih112 reviewed (changes requested):
LGTM, 

Only thing is that currently the right click menu still says: "Underline Unannotated Variables", would recommend changing this to use "Highlight" instead.
![image](https://github.com/sys-bio/AntimonyEditor/assets/82477579/714a3ae9-ea97-4739-90dc-3f0f8169000c).

evaxliu reviewed (changes requested):
Go ahead and make changes based on Edison's review and then we can proceed on creating the highlight color options

ttbt28 commented:
I already added the dropdown to choose colors of highlight, and deleted the css file so that all the tests are passed. I'm sorry, this is the first time I have used the pull request, so I did so many commits. Thanks!
